apps/assets/server/models/cloudinary.js

```diff
@@ -0,0 +1,47 @@
+const { getSecret } = require("../../helpers/secrets-management");
+const cloudinary = require('cloudinary').v2;
+
+class Cloudinary {
+  constructor(storage) {
+    this.name = storage.name;
+    this.ready = getSecret(storage.tenant, storage.authentication).then((decrypted) => {
+      const auth = decrypted.value;
+      cloudinary.config({
+        cloud_name: storage.metadata.bucketName,
+        api_key: auth.accessKeyId,
+        api_secret: auth.secretAccessKey,
+        secure: true
+      });
+    });
+  }
+
+  list(prefix = '/') {
+
+    if (prefix === '/') {
+      // load all resources
+      return cloudinary.api.resources();
```

Comment:
`/` should return only the resources on the main "folder". not all.

Reply:
fixed